cpp/include/cudf/io/parquet.hpp

```diff
@@ -699,6 +713,30 @@ class parquet_writer_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Sets the maximum page size, in bytes.
+   *
+   * @param val The page size to use.
+   * @return this for chaining.
+   */
+  parquet_writer_options_builder& max_page_size_bytes(size_t val)
```

Comment:
Understood.  This is a soft limit, since the current implementation operates a fragment-at-a-time, allowing a single fragment to over-fill a page.  I believe this is consistent with other parquet writers, such as parquet-mr.